src/components/Navbar2/Navbar2.css

```diff
@@ -240,6 +240,12 @@
     height 200ms ease 100ms, left 200ms ease-in-out 1ms;
 }
 
+@media (max-width: 991px) {
+  .dui-navbar2 .item.submenu {
+    position: fixed;
+  }
+}
+
```

Comment:
There is another `@media (max-width: 991px) {` in line 450. Can we move this in there?